docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. [Flatcar Linux Update Operator
+(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) can keep the operating system
+updated every time there is a new Flatcar release. FLUO is a supported Lokomotive component. This
+guide will show how to install it and disable its auto-update feature for specific nodes.
+
+## Prerequisites
+
+- A Kubernetes cluster accessible via `kubectl`.
+
+## Steps
+
+### Step 1: Disable auto-update on nodes
+
+If you want to update the Flatcar OS of particular nodes manually in a controlled fashion, there is
+a way to disable automatic updates. Disabling updates can come handy when the workloads run by these
+machines are storage related or ingress nodes, where you cannot afford the slightest loss of
+connection.
+
+Please annotate the nodes with the following annotation:
+
+```bash
+kubectl annotate node <node name> "flatcar-linux-update.v1.flatcar-linux.net/reboot-paused=true"
+```
+
+### Step 2: Configure FLUO
+
+Add the following content to your cluster configuration (`*.lokocfg` file):
+
+```tf
+component "flatcar-linux-update-operator" {}
+```
+
+### Step 3: Install FLUO
+
+Execute the following command to deploy the `flatcar-linux-update-operator` component:
+
+```bash
+lokoctl component apply flatcar-linux-update-operator
+```
+
+Verify the pods in the `reboot-coordinator` namespace are in the `Running` state (this may take a few
+minutes):
+
+```bash
+kubectl -n reboot-coordinator get pods
+```
```

Comment:
At the end, I'd add some testing commands like mentioned in https://github.com/kinvolk/flatcar-linux-update-operator#test so one can manually verify that the operator is functional.